src/Symfony/Component/Config/Util/XmlUtils.php

```diff
@@ -202,7 +202,7 @@ public static function phpize($value)
                 return '0x' === $value[0].$value[1] ? hexdec($value) : (float) $value;
             case preg_match('/^0x[0-9a-f]++$/i', $value):
                 return hexdec($value);
-            case preg_match('/^(-|\+)?[0-9]+(\.[0-9]+)?$/', $value):
+            case preg_match('/^(-|\+)?\d+(\.\d+)?$/', $value):
```

Comment:
whihc means your change is changing the feature, and so should be submitted separately as a feature addition (and not on 2.3)